internal/controller/bootstrap/init.go

```diff
@@ -254,10 +255,22 @@ func (b *Bootstrap) InitResources(instance *apiv3.CommonService, forceUpdateODLM
 		}
 	}
 
-	klog.Info("Installing ODLM Operator")
-	if err := b.renderTemplate(constant.ODLMSubscription, b.CSData); err != nil {
+	// Install ODLM Operator
+	// Check if CatalogSource contains the correct version of ODLM
+	// if contains, install ODLM Operator
+	// if not, skip the installation of ODLM Operator, and show warning event
+	installODLM, err := util.CheckODLMCatalogSource(b.Reader, b.CSData.ODLMCatalogSourceName, b.CSData.ODLMCatalogSourceNs)
```

Comment:
The second parameter passed in is catalog. But the function itself state the second parameter is packageName? Is there any discrepancy? 
```
func CheckODLMCatalogSource(r client.Reader, packageName, ns string) (bool, error) {
```